src/App.js

```diff
@@ -10,6 +10,7 @@ import react from "react";
 import Navbar from "components/common/Navbar/Navbar";
 import NotFoundPage from "pages/NotFoundPage/NotFoundPage";
 import LandingPage from "pages/LandingPage/LandingPage";
```

Comment:
You can create an index file in pages and export the page components from there. Here, you can do something like import {NotFoundPage, LandingPage} from 'pages'

Reply:
okay